### PR TITLE
[dev] - AB#69096: [Giving][Android/Logged In/Out]- The "New Gift" and the "Thank you Confirmation" Pages are not opening completely! there is a Gap on the top of the pages

### PR DESCRIPTION
**Pull request by pakkibokati2012**

### CODE REVIEW

File: src/modules/bottom-sheet/bottom-sheet.tsx [hunk 1]
```diff
@@ -18,7 +18,13 @@ import {LayoutUtils} from '../utilities';
 const StyledWrapper = Styled.View`
   width: 100%; height: 100%;
   backgroundColor: ${({theme}) => theme.colors.white};
-  margin-top: ${majorScale(12)}px;
+  margin-top: ${
+    StatusBar.currentHeight
```
Comment (pakkibokati2012) [inline src/modules/bottom-sheet/bottom-sheet.tsx:22]: StatuBar.currentHeight is undefined for iOS

File: src/modules/bottom-sheet/bottom-sheet.tsx [hunk 2]
```diff
@@ -117,24 +123,22 @@ export const BottomSheet: React.FC<IProps> = ({
...
           {/* Content */}
           <StyledContentWrapper>{children}</StyledContentWrapper>
-
           {/* Footer */}
           <StyledFooterWrapper>
             <SafeAreaView>{footer}</SafeAreaView>
           </StyledFooterWrapper>
         </StyledSectionWrapper>
       </StyledWrapper>
+      </SafeAreaView>
```
Comment (pakkibokati2012) [inline src/modules/bottom-sheet/bottom-sheet.tsx:141]: used SafeAreaView which works only in ios. This results in margin-top to take effect from the end of the notch for ios devices.

**Review by raoufr — APPROVED** [review]
looking good to me

**Comment by ajaybushel** [comment]
Can we please name this PR with regards to what has been changed? This will help us get a better idea of what's the PR about.

**Review by ajaybushel — APPROVED** [review]
👍

**Review by bittu1028 — APPROVED** [review]

**Pull request merged**